kratos/includes/convection_diffusion_settings.h

```diff
@@ -450,36 +450,198 @@ class ConvectionDiffusionSettings
+        {
+            rSerializer.save("SpecificHeatVar",mpSpecificHeatVar);
+        }
+        rSerializer.save("ReactionVarSaved", mpReactionVar != nullptr && mpReactionVar->Name() != "");
+        if (mpReactionVar != nullptr && mpReactionVar->Name() != "")
+        {
+            rSerializer.save("ReactionVar",mpReactionVar);
+        }
+
+        // rSerializer.save("DensityVar",mpDensityVar);
+        // rSerializer.save("DiffusionVar",mpDiffusionVar);
+        // rSerializer.save("UnknownVar",mpUnknownVar);
+        // rSerializer.save("VolumeSourceVar",mpVolumeSourceVar);
+        // rSerializer.save("SurfaceSourceVar",mpSurfaceSourceVar);
+        // rSerializer.save("ProjectionVar",mpProjectionVar);
+        // rSerializer.save("ConvectionVar",mpConvectionVar);
+        // rSerializer.save("MeshVelocityVar",mpMeshVelocityVar);
+        // rSerializer.save("TransferCoefficientVar",mpTransferCoefficientVar);
+		// rSerializer.save("VelocityVar",mpVelocityVar);
+ 		// rSerializer.save("SpecificHeatVar",mpSpecificHeatVar);
+        // rSerializer.save("ReactionVar",mpReactionVar);
+
+        rSerializer.save("mis_defined_DensityVar",mis_defined_DensityVar);
+        rSerializer.save("mis_defined_DiffusionVar",mis_defined_DiffusionVar);
+        rSerializer.save("mis_defined_UnknownVar",mis_defined_UnknownVar);
+        rSerializer.save("mis_defined_VolumeSourceVar",mis_defined_VolumeSourceVar);
+        rSerializer.save("mis_defined_SurfaceSourceVar",mis_defined_SurfaceSourceVar);
+        rSerializer.save("mis_defined_ProjectionVar",mis_defined_ProjectionVar);
+        rSerializer.save("mis_defined_ConvectionVar",mis_defined_ConvectionVar);
+        rSerializer.save("mis_defined_MeshVelocityVar",mis_defined_MeshVelocityVar);
+        rSerializer.save("mis_defined_TransferCoefficientVar",mis_defined_TransferCoefficientVar);
+        rSerializer.save("mis_defined_VelocityVar",mis_defined_VelocityVar);
+        rSerializer.save("mis_defined_SpecificHeatVar",mis_defined_SpecificHeatVar);
+        rSerializer.save("mis_defined_ReactionVar",mis_defined_ReactionVar);
 
 // 	  rSerializer.save("",);
     }
 
     virtual void load(Serializer& rSerializer)
     {
-        rSerializer.load("DensityVar",mpDensityVar);
-        rSerializer.load("DiffusionVar",mpDiffusionVar);
-        rSerializer.load("UnknownVar",mpUnknownVar);
-        rSerializer.load("VolumeSourceVar",mpVolumeSourceVar);
-        rSerializer.load("SurfaceSourceVar",mpSurfaceSourceVar);
-        rSerializer.load("ProjectionVar",mpProjectionVar);
-        rSerializer.load("ConvectionVar",mpConvectionVar);
-        rSerializer.load("MeshVelocityVar",mpMeshVelocityVar);
-        rSerializer.load("TransferCoefficientVar",mpTransferCoefficientVar);
-        rSerializer.load("VelocityVar",mpVelocityVar);
-		rSerializer.load("SpecificHeatVar",mpSpecificHeatVar);
-        rSerializer.load("ReactionVar",mpReactionVar);
+        bool denstityvartoload;
```

Comment:
please check the [Style Guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide) for how to name variables => `load_density_variable`